MantisPhpClient.php

```diff
@@ -347,15 +363,16 @@ private function getProjectFromProjectTree( $p_project_id, $p_projects ) {
     /**
      * Checks if a user exists.
      *
-     * @param $p_user array that can contain id, name, or email.
+     * @param StdClass $p_user array that can contain id, name, or email.
      * @return true found, false otherwise.
+     * @throws Exception
      */
     public function userExists( $p_user ) {
         $t_mantis_version = $this->getMantisVersion();
 
         if ( version_compare( $t_mantis_version, '1.2.16dev' ) >= 0 ) {
             try {
-                $this->soap_client->mc_project_get_issues_for_user( $this->username, $this->password, ALL_PROJECTS, 'reported', $p_user, 1, 1 );
+                $this->soap_client->mc_project_get_issues_for_user( $this->username, $this->password, 'ALL_PROJECTS', 'reported', $p_user, 1, 1 );
```

Comment:
Remove the quotes around ALL_PROJECTS, and define it as a constant with value 0.